src/sentry/static/sentry/app/views/performance/trends/utils.tsx

```diff
@@ -32,54 +39,67 @@ export const DEFAULT_MAX_DURATION = '15min';
 
 export const TRENDS_FUNCTIONS: TrendFunction[] = [
   {
-    label: 'Duration (p50)',
+    label: 'p50',
     field: TrendFunctionField.P50,
     alias: 'percentile_range',
     chartLabel: 'p50()',
     legendLabel: 'p50',
+    aggregation: 'p50',
```

Comment:
I think this key needs to be added to the type definition of `TrendFunction` and if it's added as `AggregateKey`, maybe you can skip the cast from `string` in `generateTrendFunctionAsString`